Jenkinsfile-smoke.groovy

```diff
@@ -0,0 +1,23 @@
+/*
+ * Requires: https://github.com/RedHatInsights/insights-pipeline-lib
+ */
+
+@Library("github.com/RedHatInsights/insights-pipeline-lib") _
+
+
+// this 'if' statement makes sure this is a PR, so we don't run smoke tests again
+// after code has been merged into the stable branch.
+if (env.CHANGE_ID) {
+    runSmokeTest (
+        // the service-set/component for this app in e2e-deploy "buildfactory"
+        ocDeployerBuilderPath: "topological-inventory/topological-inventory-api",
+        // the service-set/component for this app in e2e-deploy "templates"
+        ocDeployerComponentPath: "topological-inventory/topological-inventory-api",
+        // the service sets to deploy into the test environment
+        ocDeployerServiceSets: "platform-mq, sources, topological-inventory",
```

Comment:
You'll need to remove the white space: `platform-mq,sources,topological-inventory`

Reply:
Fixed